app/src/main/res/values-es/strings.xml

```diff
@@ -9,8 +9,10 @@
     <string name="title_network_overview">Red {interface_name}</string>
     <string name="title_device_detail">Dispositivo {deviceIp}</string>
     <string name="title_card_device_info">Información del dispositivo</string>
+    <string name="field_name_device_name">Device Name</string>
```

Comment:
If the translation is not known, just remove these fields from the translation.

That way, it's easier to find missing translations when somebody wants to resolve them